html/janus.js

```diff
@@ -609,11 +600,9 @@ function Janus(gatewayCallbacks) {
 			Janus.debug("Got an ack on session " + sessionId);
 			Janus.debug(json);
 			var transaction = json["transaction"];
-			if(transaction !== null && transaction !== undefined) {
+			if(transaction) {
 				var reportSuccess = transactions[transaction];
-				if(reportSuccess !== null && reportSuccess !== undefined) {
-					reportSuccess(json);
-				}
+				if(reportSuccess) reportSuccess(json);
```

Comment:
Please put the action on a different line and indented (code style). The same seems to be happening in a couple of other instances, so please fix it there too.